index.html

```diff
@@ -0,0 +1,153 @@
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">CC Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/chat.png" alt="lecture image">
+                    <p class="t1">Forum</p>
+                </div>
+                <p class="program-disc">Have questions, drop us as chat.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/workshop.png" alt="lecture image">
+                    <p class="t1">Workshop</p>
+                </div>
+                <p class="program-disc">We hold workshops, so no one is left behind.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/meeting.png" alt="lecture image">
+                    <p class="t1">CC ignite</p>
+                </div>
+                <p class="program-disc">We have weekly meetings to address challenges faced.</p>
+            </div>
+        </div>
+        <div class="button">
+            <a class="start-learning" href="#">Start Learning</a>
+        </div>
+        <p class="whole-program font-face">SEE THE WHOLE PROGRAM</p>
+    </section>
+    
+    <!-- speaker section -->
+    <section class="container main-speakers">
+        <div class="p-cont">
+            <p class="program-title">Featured Speakers</p>
+            <hr>
+        </div>
+
+        <div id="dynamic-page" class="row"></div>
+        <div id="more" class="row grid"></div>
+      
+        <div class="read-button">
+            <button onclick="ReadMore()" class="read-more" id="toggle">MORE<i class="fa fa-angle-down fa-lg"></i></button>
+        </div>
+    </section>
+
+    <!-- partner section -->
+    <section class="partner">
+        <div class="p-cont">
+            <p class="program-title">Partner</p>
+            <hr>
+        </div>
+        <div class="partner-imgs">
+            <img src="images/absa-logo-round.svg" class="absa" alt="absa">
+            <img src="images/vodacom-logo.png" class="vodacom" alt="vodacom logo">
+            <img src="images/googlelogo_color_92x30dp.png" alt="google" class="google">
+            <img src="images/microsofr.png" alt="microsoft" class="micro">
+            <img src="images/aws_smile-header-mobile-en-white_48x29@2x.png" alt="aws" class="aws">
+        </div>
+    </section>
+
+    <!-- footer -->
+    <footer>
+        <p class="f-logo">CodeWorld</p>
+        <div class="footer-text">
+            <span>2022 CodeWorld, some rights reserved.</span>
+        </div>
+    </footer>
```

Comment:
- [ ] [_OPTIONAL_] your implementation of the footer is a bit different than that of the design, kindly fix that.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/164548069-d7097e81-6389-457d-a824-6b7132eaec07.png)

### your implementation
![image](https://user-images.githubusercontent.com/59416136/164548156-1b55e45a-af3b-43d8-aed2-0af6b98f8ff6.png)

Please try to reduce the space.